docs/DeveloperGuide/Layer/Usecase/TrafficSimulation/Abstract/index.md

```diff
@@ -0,0 +1,121 @@
+    NPC Vehicle Simulator --> Ego Vehicle:Get Transform
+    NPC Vehicle Spawner --> NPC Vehicles:Spawn/Despawn
+    NPC Vehicle Simulator --> NPC Vehicles:Set Position and Rotation
+    Ramdom Traffic Simulator --> Environment Components:Get Data
+    class Ramdom Traffic Simulator{
+        - NPC Vehicle Spawner
+        - NPC Vehicle Simulator
+        + Get Data()
+    }
+    class NPC Vehicle Spawner{
+        + Spawn/Despawn()
+    }
+    class NPC Vehicle Simulator{
+        - Cognition Step
+        - Decision Step
+        - Control Step
+        - Visualization Step
+        + Get Transform()
+        + Set Position and Rotation()
+    }
+    class Ego Vehicle{
+    }
+    class NPC Vehicles{
+        + NPC 1
+        + NPC 2
+        ...
+    }
+    class Environment Components{
+        + Traffic Light
+        + Traffic Lane
+        + Traffic Intersection
+        + Stop Line
+    }
+```
+
+## Configuration
+`Traffic Simulation` can be configured from `TrafficSimulator` component.
+
+The configurable elements are listed in the following table:
+
+### General Settings
+
+![Configulations0](./config0.png)
+
+| Parameter | Description |
+|---|---|
+| Ego Vehicle | Ego vehicle handler. If not set, the manager creates a dummy ego<br>This reference is also set automatically when the Ego spawns via the traffic simulator |
+| Seed | Seed value for random generator |
+| Traffic Intersections | The field that is set `TrafficIntersection` objects<br>`TrafficIntersection` to be set is controlled by `Traffic Simulation` |
+
+### NPC Vehicle Settings
+
+![Configulations1](./config1.png)
+
+| Parameter | Description |
+|---|---|
+| Vehicle Config | Parameters for NPC vehicle control<br/>`Sudden Deceleration` is a deceleration related to emergency braking |
+| Obstacle Layer Mask | The obstacle layer for raytracing the collision distances |
+| Ground Layer Mask | The Ground layer for raytracing the collision distances |
+| Max Vehicle Count | The maximum number of vehicles that can simultaneously live in the scene<br>Lowering this value results in less dense traffic but improves the simulator's performance |
+| Spawn Distance To Ego | The minimal distance between the EGO and the NPC to spawn |
+
+### Debug
+
+![Configulations2](./config2.png)
+
+| Parameter | Description |
+|---|---|
+| Show Gizmos | Enable the checkbox to show all visualization using editor gizmos |
+| Show Yielding Phase | Enable the checkbox to show editor gizmos that visualize `Yielding Phase` of NPCs |
+| Show Obstacle Checking | Enable the checkbox to show editor gizmos that visualize `Obstacle Checking` phase of NPCs |
+| Show Spawn Points | Enable the checkbox to show editor gizmos that visualize `Spawn Points` where NPCs is generated |
+
+### Random Traffic Sims
+
+![Configulations3](./config3.png)
+
+| Parameter | Description |
+|---|---|
+| Enable Simulation | Enable the checkbox to `Random Traffic Sims` which spawn NPC vehicles randomly |
```

Comment:
Use periods in English sentences.

Reply:
Fixed it in [#2299ea8](https://github.com/tier4/AWSIM/pull/416/commits/2299ea8ebd5cdebad613b5e224c5e53f11a7b4a5).